optimum/onnxruntime/modeling_ort.py

```diff
@@ -1115,6 +1125,9 @@ def forward(
                 # TODO: This allows to support sentence-transformers models (sentence embedding), but is not validated.
                 last_hidden_state = next(iter(model_outputs.values()))
 
+        if return_dict:
+            return model_outputs
```

Comment:
can you please add this to other ORModelForxxx, it's strange that we never handled this argument 🤔